src/style.css

```diff
@@ -0,0 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap&subset=cyrillic");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.menu {
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.navigation {
+  display: flex;
+
+}
+
+.logo a {
+  display: flex;
+  padding: 10px 0;
+}
+
+.navbar {
+  display: flex;
+  margin-left: auto;
+  align-items: center;
+}
+
+.navbar .nav-link {
+  margin-right: 20px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000;
+  padding: 23px 0 22.5px;
+}
+
+.navbar .nav-link:last-child {
+  margin-right: 0;
+}
+
+.navbar .navLink:hover {
+  color: #00acdc;
+}
+
+.navbar .is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.navbar .is-active::after {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element